src/stylesheets/main.scss

```diff
@@ -23,13 +23,14 @@
 // 4. Layout-related sections
 @import
   'layout/header',
+  'layout/first',
   'layout/about-us',
   'layout/our-services',
   'layout/portfolio',
   'layout/pracing-plan',
   'layout/team',
   'layout/blog',
-  'layout/contact-us',
+  //'layout/contact-us',
```

Comment:
Why did you commented it?

Reply:
File layout/_contact-us missing

Reply:
> File layout/_contact-us missing

Ok, it's my fault, probably I forgot to add a new file, you can do it instead of me.